### PR TITLE
CRIMRE-320 Increase user management pages size to 50

### PR DESCRIPTION
**Pull request by timpeat**
## Description of change

Increases manage user pagination to 50 and simplifies return path to page 1 on all actions. 

(New design will remove need for returning to other pages on success.)

Adds a "set_flash" method for accessing namespace flash messages.

### CODE REVIEW

File: app/controllers/admin/manage_users_controller.rb [hunk 1]
```diff
@@ -7,12 +7,9 @@ def text_namespace
     end
 
     before_action :require_user_manager!
-    before_action :set_return_url, only: [:edit, :update]
 
     def index
-      page = (index_params[:page].presence || 1)
-      per_page = Rails.configuration.x.admin.pagination_per_page
-      @users = User.all.order(first_name: :asc, last_name: :asc).page(page).per(per_page)
+      @users = User.all.order(first_name: :asc, last_name: :asc).page(params[:page])
```
Comment (willmcb) [inline app/controllers/admin/manage_users_controller.rb:12]: Could we make this multi-line - just a bit easier to parse :)

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -31,4 +31,28 @@ def set_search(filter: ApplicationSearchFilter.new, sorting: {})
       filter:, sorting:, pagination:
     )
   end
+
+  # Sets the full flash message based on the message key.
```
Comment (willmcb) [inline app/controllers/application_controller.rb:35]: 👍

File: config/locales/en/manage_users.yml [hunk 3]
```diff
@@ -8,6 +8,17 @@ en:
...
+    flash:
+      success:
+        title: Success
+        new_user_created: Email address has been added
+        user_updated: Email address has been updated
+      important:
+        title: Important
+        cannot_access:
+          - You do not have access to that page
+          - Contact laa-crime-apply@digital.justice.gov.uk if you think this wrong
```
Comment (willmcb) [inline config/locales/en/manage_users.yml:20]: I think there is a missing 'is' at the end here :)

**Review by willmcb — APPROVED** [review]
LGTM 👍

File: app/controllers/admin/manage_users_controller.rb (hunk 1 above)
Comment (timpeat) [reply app/controllers/admin/manage_users_controller.rb]: Done :)

File: app/controllers/application_controller.rb [hunk 4]
```diff
@@ -31,4 +31,28 @@ def set_search(filter: ApplicationSearchFilter.new, sorting: {})
...
+  # Like the AppTextHelper#named_text, it will scope locales according
+  # to the text_namespace set on the controller. If none is set, it will
+  # use root.
+  #
+  # Defaults to "success: true", pass option "success: false" to override.
+  #
+  # Any other options will be passed to I18n.translate.
+  #
+  def set_flash(message_key, options = {})
+    success = options.fetch(:success, true)
```
[the remark this answers is not in the document]
Comment (timpeat) [reply app/controllers/application_controller.rb:46]: The default is to assume success is true. If set as success: false it will return false.

**Pull request merged**